test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```
@@ -130,6 +130,18 @@ spec:
       openAPIV3Schema:
         type: boolean
         default: false
+  - name: kubeControlPlaneLogLevel
+    schema:
+      openAPIV3Schema:
+        type: string
+        description: "Log level for kube-apiserver, kube-scheduler and kube-controller-manager"
+        default: "0"
+  - name: kubeletLogLevel
+    schema:
+      openAPIV3Schema:
+        type: string
+        description: "Log level for kubelets on control plane and worker nodes"
+        default: "0"
```

NIT!

```suggestion
  - name: kubeControlPlaneLogLevel
    schema:
      openAPIV3Schema:
        type: string
        description: "Log level for kube-apiserver, kube-scheduler and kube-controller-manager"
        default: ""
  - name: kubeletLogLevel
    schema:
      openAPIV3Schema:
        type: string
        description: "Log level for kubelets on control plane and worker nodes"
        default: ""
```

I'd suggest to default to empty string and not patch if the value is empty string (`enabledIf`). This way we would always keep the default. (may require to split the patch with regards to the variables)

---

Good idea!